idl/partition_checksum.idl.hh

```diff
@@ -103,3 +103,22 @@ enum class repair_row_level_start_status: uint8_t {
 struct repair_row_level_start_response {
     repair_row_level_start_status status;
 };
+
+enum class removenode_cmd : uint32_t {
+     prepare,
+     sync_data,
+     abort,
+     error,
+     ok,
+};
+
+struct removenode_request {
+    removenode_cmd cmd;
+    utils::UUID req_id;
+    std::list<gms::inet_address> leaving_nodes;
+    std::list<gms::inet_address> skip_sync_on_nodes;
+};
+
+struct removenode_response {
+    removenode_cmd cmd;
+};
```

Comment:
Ah, I see you use it as a response. Better to have its own enum for that.

Reply:
I prefer a single cmd for both request and response to make the protocol simple. We have similar usage in row level repair.

Reply:
It is less to type, but confusing to readers. The request and response are not the same thing.

Reply:
I still think sending a command as a response is confusing.